docs/pages/includes/metrics.mdx

```diff
@@ -1,59 +1,67 @@
+| `gcs_event_storage_downloads_seconds`                | histogram   | GCP GCS             | Latency for GCS download operations.                                                               |
+| `gcs_event_storage_downloads`                        | counter     | GCP GCS             | Number of downloads from the GCS backend.                                                          |
+| `gcs_event_storage_uploads_seconds`                  | histogram   | GCP GCS             | Latency for GCS upload operations.                                                                 |
+| `gcs_event_storage_uploads`                          | counter     | GCP GCS             | Number of uploads to the GCS backend.                                                              |
+| `grpc_server_started_total`                          | counter     | Teleport Auth       | Total number of RPCs started on the server.                                                        |
+| `grpc_server_handled_total`                          | counter     | Teleport Auth       | Total number of RPCs completed on the server, regardless of success or failure.                    |
+| `grpc_server_msg_received_total`                     | counter     | Teleport Auth       | Total number of RPC stream messages received on the server.                                        |
+| `grpc_server_msg_sent_total`                         | counter     | Teleport Auth       | Total number of gRPC stream messages sent by the server.                                           |
+| `heartbeat_connections_received_total`               | counter     | Teleport Auth       | Number of times the Auth Service received a heartbeat connection.                                  |
+| `s3_requests_total`                                  | counter     | Amazon S3           | Total number of requests to the S3 API.                                                            |
+| `s3_requests`                                        | counter     | Amazon S3           | Total number of requests to the S3 API grouped by result.                                                        |
+| `s3_requests_seconds`                                | histogram   | Amazon S3           | Request latency for the S3 API.                                                                    |
+| `teleport_audit_emit_events`                         | counter     | Teleport Audit Log  | Number of audit events emitted.                                                                    |
+| `teleport_audit_athena_batch_processing_seconds`     | histogram   | Teleport Audit Log  | Duration of processing single batch of events in athena audit log.                                 |
+| `teleport_audit_athena_s3_flush_seconds`             | histogram   | Teleport Audit Log  | Duration of flushing parquet files to s3 in athena audit log.                                      |
```

Comment:
Metric names are hard to change because people rely on them once they have dashboards, alerts, etc.

Should we try to come up with a name that's not specific to Athena or S3 (so that when we get this working on prem with alternate technologies the metric names still make sense)?

Reply:
for prefix we can use `teleport_audit_presto_`? or `teleport_audit_objectstorage_`? 

or we can go with more generic names like: `teleport_audit_batch_processing_time` and `teleport_audit_data_flush_time`, and use description here to indicate that so far they are used only in athena based logger. What do you think @zmb3 ?

Reply:
That works for me.

Reply:
@rosstimothy  @zmb3 right now i think that going that generic as `teleport_audit_data_flush_time` may not be best option. You can configure multiple events logger, and if we go with very generic name like `teleport_audit_data_flush_time` and someone uses two loggers that use the same metrics, data will be invalid because both loggers will write to the same metric. (because they are not vector labels).

So what we can do is either introduce that new generic metric as vector with label as 'athena' for now, or go with `teleport_audit_presto_` or keep `teleport_audit_athena_`.

Reply:
@zmb3 @rosstimothy ☝️

Reply:
I don't have a strong preference and leave it up to you to decide on names that make the most sense. You are more familiar with the subject matter than I am :-)

Reply:
i went with `parquetlog`